flink-runtime/src/test/java/org/apache/flink/runtime/state/StateBackendMigrationTestBase.java

```diff
@@ -1231,13 +1239,14 @@ public void testStateMigrationAfterChangingTTLFromEnablingToDisabling() {
             testKeyedValueStateUpgrade(initialAccessDescriptor, newAccessDescriptorAfterRestore);
             fail("should have failed");
         } catch (Exception expected) {
-            Assert.assertTrue(
-                    ExceptionUtils.findThrowable(expected, StateMigrationException.class)
-                            .isPresent());
+            assertThat(
+                            ExceptionUtils.findThrowable(expected, StateMigrationException.class)
+                                    .isPresent())
+                    .isTrue();
         }
     }
 
-    @Test
+    @TestTemplate
     public void testStateMigrationAfterChangingTTLFromDisablingToEnabling() {
```

Comment:
```suggestion
    void testStateMigrationAfterChangingTTLFromDisablingToEnabling() {
```
could be package private